src/uct/tcp/tcp_iface.c

```diff
@@ -81,10 +81,29 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
                                 ucs_offsetof(uct_tcp_iface_config_t, rx_mpool), ""),
 
   {"PORT_RANGE", "0",
-   "Generate a random TCP port number from that range. A value of zero means\n "
+   "Generate a random TCP port number from that range. A value of zero means\n"
    "let the operating system select the port number.",
    ucs_offsetof(uct_tcp_iface_config_t, port_range), UCS_CONFIG_TYPE_RANGE_SPEC},
 
+#if UCT_TCP_EP_KEEPALIVE
+  {"KEEPIDLE", "10s",
```

Comment:
how would it correlate with `UCX_KEEPALIVE_TIMEOUT` ? Does it mean it is always enabled for TCP?

Reply:
it could not correlate: ep_check functions are called sequenced, by portions, kernel-based keepalive packets may be called simultaneously & affect to overall performance

Reply:
next PR: 
- add uct_iface_attr_t capability: UCT_IFACE_FLAG_KEEPALIVE - it means keepalive_interval field is not ignored
- "auto" will mean to take new parameter (keepalive_interval) passed to uct_ep_create as new field in uct_ep_arams